docs/topics/proc_running-the-secured-booster-integration-tests-on-sb-vx.adoc

```diff
@@ -1,32 +1,59 @@
 
-[#running-the-secured-booster-integration-tests-on-vertx]
-= Running the {name-mission-secured} Booster Integration Tests on {VertX}
+[#running-the-secured-booster-integration-tests-on-{parameter-runtime}]
+= Running the {name-mission-secured} Booster Integration Tests on {runtime}
+//TODO: specify parameter in master file
 
 .Prerequisites
 
+
 * The `oc` client authenticated.
+// Should probably include the prerequisite to create a separate project for IT's in OpenShift
```

Comment:
If the process requires it, I certainly concur.

Reply:
IIRC, it used to be the case that running integration tests inside an OpenShift project already containing a booster application resulted in the removal of this pre-existig application once the test were done. If that still apllies, using a separate project to run the IT's could  help  avoid accidental deletion of a running application. We could either include a step to create an extra project or include a note about it to alert the users to this possibility. @Ladicek WDYT?

Reply:
So, if all the boosters migrated their tests to Arquillian Cube, there would be a very simple answer: verify that you have permissions to create new projects in OpenShift and add `-Dnamespace.use.current=false -DenableImageStreamDetection=true` when running the tests. With these parameters, Arquillian Cube would automatically create an ephemeral OpenShift project, run the tests in it and then destroy it.

Unfortunately, some boosters still use OpenShiftTestAssistant for their tests :-/

I think adding a note about the possibility to run the tests in an extra namespace is a good idea.